src/image-display.cc

```diff
@@ -120,6 +120,23 @@ static ExifImageOp GetExifOp(Magick::Image &img) {  // NOLINT
     return {};
 }
 
+// An extended version of Magick::readImages that requests a
+// decoding/raster with a transparent background.
+static void readImagesWithTransparentBackground(
```

Comment:
One sentence how we're doing that is probably good: by priming the opacity in the image info.

Reply:
(solved)